gwswlib/threedi.py

```diff
@@ -181,38 +282,33 @@ def get_mapping_value(self, mapping, hydx_value, record_code, name_for_logging):
             )
             return None
 
-    def get_code(self, connection, default_code=""):
-        """
-        Args:
-            code1(string): object code
-            code2(string): object code 2
-            default_code: returned value when code is None or ''
-
-        Returns:
-            (string): combined area code
-        """
+    def check_if_nodes_of_connection_exists(self, connection, default_code=""):
         connection_code = connection.identificatieknooppuntofverbinding
         code1 = connection.identificatieknooppunt1
         code2 = connection.identificatieknooppunt2
 
         manh_list = [manhole["code"] for manhole in self.manholes]
         if code1 not in manh_list:
             logging.error(
-                "Connection node %r could not be found for record %r",
+                "Start connection node %r could not be found for record %r",
                 code1,
                 connection_code,
             )
-            if code1 is None or code1 == "":
-                code1 = default_code
         if code2 not in manh_list:
             logging.error(
-                "Connection node %r could not be found for record %r",
+                "End connection node %r could not be found for record %r",
                 code2,
                 connection_code,
             )
-            if code2 is None or code2 == "":
-                code2 = default_code
-        element_codes = code1 + "-" + code2
+
+    def get_connection_display_name_from_manholes(self, connection, default_code=""):
+        code1 = connection.identificatieknooppunt1
+        code2 = connection.identificatieknooppunt2
+
+        if code1 is None or code1 == "":
```

Comment:
Kan deze ook?

```suggestion
        if not code1:
```

Dit kan trouwens niet als de code ook (legaal) `0` mag zijn.
Wordt de lege string trouwens niet elders al afgevangen en als `None` erin gezet? Dan zou een `if code1 is None` voldoende zijn.